views/js/review/plugins/navigation/review-panel/plugin.js

```diff
@@ -21,16 +21,35 @@
 define([
     'core/promiseTimeout',
     'taoTests/runner/plugin',
-    'taoReview/review/plugins/navigation/review-panel/panel',
-    'taoReview/review/plugins/navigation/review-panel/panel-data'
+    'taoReview/review/services/navigation-data',
+    'taoReview/review/plugins/navigation/review-panel/panel'
 ], function (
     promiseTimeout,
     pluginFactory,
-    reviewPanelFactory,
-    reviewPanelService
+    navigationDataServiceFactory,
+    reviewPanelFactory
 ) {
     'use strict';
 
+    const filters = {
+        /**
+         * No filter, keep all items
+         * @returns {Boolean}
+         */
+        all() {
+            return true;
+        },
+
+        /**
+         * Filter for incorrect items
+         * @param {mapEntry} item
+         * @returns {Boolean}
+         */
+        incorrect(item) {
+            return !item.informational && (!item.maxScore || item.score !== item.maxScore);
```

Comment:
If the item doesn't have maxScore (for example write text or send file) it will be considered as incorrect?

Reply:
Yes indeed. Without the max score we cannot know if the item is correct or no. So incorrect by default.